pkg/custompluginmonitor/types/config_test.go

```diff
@@ -0,0 +1,249 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	"k8s.io/node-problem-detector/pkg/types"
+)
+
+func TestCustomPluginConfigApplyConfiguration(t *testing.T) {
+	globalTimeout := 6 * time.Second
+	globalTimeoutString := globalTimeout.String()
+	invokeInterval := 31 * time.Second
+	invokeIntervalString := invokeInterval.String()
+	maxOutputLength := 79
+	concurrency := 2
+
+	ruleTimeout := 1 * time.Second
+	ruleTimeoutString := ruleTimeout.String()
+
+	utMetas := []struct {
```

Comment:
Use a map, add description for each test case https://github.com/kubernetes-incubator/cri-containerd/blob/master/pkg/server/helpers_test.go#L29.

Or else it's very hard to figure out what each case is testing in the future. And please do this for the other unit tests you added.

Reply:
Will do.